qa/tasks/caasp.py

```diff
@@ -0,0 +1,100 @@
+'''
+Task that deploys a CAASP cluster on all the nodes
+Linter:
+    flake8 --max-line-length=100
+'''
+import logging
+from teuthology.misc import (
+    all_roles_of_type,
+    append_lines_to_file,
+    get_file
+    )
+from teuthology.task import Task
+from util import (
+    get_remote_for_role,
+    )
```

Comment:
Could you please replace this with oneliner:
```
from tasks.util import get_remote_for_role
```
It is not necessary to use mulitple import syntax when you have just one. Also sometime on py3 it cannot find 'util' so please use for full path to the module `tasks.util`.

Reply:
This was because I cleaned up unused imports.
I changed to the onleiner.